frontend/lib/jitsi-modules/Tracks.js

```diff
@@ -44,25 +44,24 @@ const tracksRepository = () => {
   };
 
   const syncSessionStorageTrack = async (track, user) => {
-    const trackType = track.getType();
-
     if (!user) {
       user = JSON.parse(sessionStorage.getItem('user'));
     }
 
-    if (user && track.isLocal()) {
+    const seat = seatsRepository.getSeat();
+
+    if (user && seat > 0 && track.isLocal()) {
```

Comment:
This is the real fix, we need to take into account if the user is on a seat, otherwise we shouldn't bother muting or unmuting.